core/arch/arm/plat-stm32mp1/scmi_server.c

```diff
@@ -422,7 +499,7 @@ int32_t plat_scmi_clock_rates_array(unsigned int channel_id,
 	if (!clock)
 		return SCMI_NOT_FOUND;
 
-	if (!stm32mp_nsec_can_access_clock(clock->clock_id))
+	if (!nsec_can_access_resource(clock->fw_id))
```

Comment:
Again, not sure we need to protect clock operations

Reply:
See my response above.